Orange/data/dask.py

```diff
@@ -1,13 +1,29 @@
 import contextlib
 import pickle
+import warnings
+from numbers import Integral
 
 import h5py
 import dask
 import dask.array as da
 import numpy as np
 import pandas
 
-from Orange.data import Table
+from Orange.data import Table, RowInstance, Instance
+
+
+class DaskRowInstance(RowInstance):
+
+    def __init__(self, table, row_index):
+        """
+        Construct a data instance representing the given row of the table.
+        """
+        warnings.warn("Creating Instances from DaskTables is inefficient", stacklevel=2)
```

Comment:
If we raise warnings every time someone uses this class we should make it clear it exists only for fallback/compatibility reasons. In that case, I would prefer Orange's warnings that fail in tests so we can make sure our existing and new code will not be using this. Intended uses should be covered in tests with explicit assertWarns.